bootloader/src/pyi_pythonlib.c

```diff
@@ -43,6 +43,106 @@
 #include "pyi_utils.h"
 #include "pyi_python.h"
 #include "pyi_win32_utils.h"
+#include "pyi_pythonlib.h"
+#include "pyi_launch.h"
+
+
+/*
+ * Look for the archive identified by path into the ARCHIVE_STATUS pool archive_pool.
+ * If the archive is found, a pointer to the associated ARCHIVE_STATUS is returned
+ * otherwise the needed archive is opened and added to the pool and then returned.
+ * If an error occurs, returns NULL.
+ *
+ * Having several archives is useful for sharing binary dependencies with several
+ * executables (multipackage feature).
+ */
+static ARCHIVE_STATUS *
+_get_archive(ARCHIVE_STATUS *archive_pool[], const char *path)
```

Comment:
This function is required in the child, but my c isn't that good, so I was unable to call this function from it's original location.